src/web/forms/fields/AutoCalc/index.js

```diff
@@ -20,20 +20,137 @@ import React from 'react';
 import { connect } from 'react-redux';
 import _get from 'lodash.get';
 import functions from './functions';
+import classNames from 'classnames';
+
+const alertFaIconClasses = {
+  danger: 'warning',
+  warning: 'warning',
+  success: 'check',
+};
 
 export const AutoCalcComponent = ({
   value,
   style,
   precision,
+  alerts,
+  // type = 'text',
```

Comment:
Please don't commit unnecessary line of code

Reply:
fixed